sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -569,3 +570,29 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    [System.Text.Json.Serialization.JsonConverterAttribute(typeof(Azure.Messaging.CloudEventConverter))]
+    public partial class CloudEvent
+    {
+        public CloudEvent(string source, string type, object jsonSerializable, System.Type? dataSerializationType = null) { }
+        public CloudEvent(string source, string type, System.ReadOnlyMemory<byte> data, string dataContentType) { }
+        public System.BinaryData? Data { get { throw null; } }
+        public string? DataContentType { get { throw null; } set { } }
+        public string? DataSchema { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object?> ExtensionAttributes { get { throw null; } }
+        public string? Id { get { throw null; } set { } }
+        public string? Source { get { throw null; } set { } }
+        public string? Subject { get { throw null; } set { } }
+        public System.DateTimeOffset? Time { get { throw null; } set { } }
+        public string? Type { get { throw null; } set { } }
+        public static Azure.Messaging.CloudEvent[]? Parse(string requestContent) { throw null; }
```

Comment:
Have a little concern here: `Parse` sounds a bit general while only JSON encoded events are allowed for the `requestContent` parameter. What if later we support binary format?

Reply:
Binary mode would likely have an HttpRequest as the input parameter rather than a string. I do think we can clarify the parameter name as `jsonContent`. One other thought is that if we ever support the Avro format, the input there would likely be a string so by using the generic `Parse` name we wouldn't be able to have a symmetrical Parse for Avro (as the signatures would be the same). I'm not sure how likely that is though so it may be okay to just stick with Parse, and we can add a ParseAvro in the future if needed.